sonic-xcvrd/scripts/xcvrd_utilities.py

```diff
@@ -0,0 +1,268 @@
+#!/usr/bin/env python2
```

Comment:
This is a library, and is not meant to be executed. Shebang is not applicable. Please remove.

Reply:
fixed